plugins/ltac/extratactics.mlg

```diff
@@ -1110,6 +1110,10 @@ TACTIC EXTEND optimize_heap
 | [ "optimize_heap" ] -> { tclOPTIMIZE_HEAP }
 END
 
+VERNAC COMMAND EXTEND infoH CLASSIFIED AS QUERY
+| ![ proof_query ] [ "infoH" tactic(tac) ] -> { fun ~pstate -> Internals.infoH ~pstate tac }
```

Comment:
I suggest giving the new command an upper case name, following the command naming convention.

Reply:
@jfehrle this is mostly an internal API change, the command must keep the old name so users are not broken by this change.